txn.go

```diff
@@ -395,6 +395,32 @@ func (tr *transactionRunner) RunTransaction(transaction *Transaction) error {
 	}
 	start := tr.clock.Now()
 	runner := tr.newRunner()
+	if len(testHooks) > 0 {
+		if sstxnRunner, ok := runner.(*sstxn.Runner); ok {
```

Comment:
reduce indents by doing one if?

Reply:
Nah, don't want the cost of the type assertion for non-test executions.